var/spack/repos/builtin/packages/perl-datetime/package.py

```diff
@@ -10,10 +10,31 @@ class PerlDatetime(PerlPackage):
     """DateTime - A date and time object for Perl"""
 
     homepage = "https://metacpan.org/pod/DateTime"
-    url = "https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-1.63.tar.gz"
+    url = "https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-1.65.tar.gz"
+
+    maintainers("EbiArnie")
 
     license("Artistic-2.0")
 
-    version("1.63", sha256="1b11e49ec6e184ae2a10eccd05eda9534f32458fc644c12ab710c29a3a816f6f")
+    version("1.65", sha256="0bfda7ff0253fb3d88cf4bdb5a14afb8cea24d147975d5bdf3c88b40e7ab140e")
```

Comment:
Is there a reason for removing `1.63`?  Usually we recommend deprecating versions for a release cycle (see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions).

Reply:
No, in this case there isn't. I added it back in as deprecated.